### ISSUE TITLE
overall product rating

### ISSUE THREAD

**Issue by krebbi** [wardhache/community_store_reviews#6]
Hi,

any uncomplicated idea how to get the products overall rating into the product list?

**Comment by wardhache** [comment]
Hi,

First of all, while testing some stuff for your question, a bug came out.
This is fixed in Pull Request #7, so make sure you get the changes. You don't need to update your package.

Then, to get the average rating in the community_product or community_product_list block is quite easy. This is what you do:

In the `view.php` of the template of the block, add the following lines (to include a css file and use the Review modal) at the beginning of the file like this:
```
defined('C5_EXECUTE') or die(_("Access Denied."));
$view->addHeaderItem('<link rel="stylesheet" href="/packages/community_store_reviews/css/community-store-reviews.css" />');
use \Concrete\Package\CommunityStoreReviews\Src\CommunityStore\Review\Review as StoreReview;
```
Then, on the preferred location within this template, add this code to get the average *approved* rating of the product (a star rating):
```
<div class="store-rating-box-small">
     <div class="store-rating" style="width: <?= StoreReview::getAverageApprovedRatingOfProduct($product); ?>%;"></div>
</div>
```
If you want to get the total average rating, and not only approved, you can use `StoreReview::getAverageRatingOfProduct($product);`. 
Just make sure you set the product object as the parameter.

I hope this helps you out.

On a side note, I will clean up this package and test it more in the near future, so we can go to version 1.0, if any problems occur while you use it, feel welcome to give me a heads-up.

Good luck!
wardhache - webhapp

**Comment by krebbi** [comment]
works like a charme! thx!
